### ISSUE TITLE
NPE while entity riding boat

### ISSUE THREAD

**Issue by GoldenGamerLP** [Minestom/Minestom#1906]
*Description*:

So, when summoning a boat and adding an entity as a passenger (in my case a player) to the boat, and try to teleport it, it prints out an exception.

*Exception*:

```
java.lang.NullPointerException: Cannot invoke "net.minestom.server.entity.Entity.getEntityMeta()" because "vehicle" is null
	at net.minestom.server.listener.PlayerVehicleListener.boatSteerListener(PlayerVehicleListener.java:39)
	at net.minestom.server.listener.manager.PacketListenerManager.processClientPacket(PacketListenerManager.java:90)
	at net.minestom.server.entity.Player.lambda$interpretPacketQueue$18(Player.java:1857)
	at org.jctools.queues.MpscUnboundedXaddArrayQueue.drain(MpscUnboundedXaddArrayQueue.java:312)
	at net.minestom.server.entity.Player.interpretPacketQueue(Player.java:1857)
	at net.minestom.server.entity.Player.update(Player.java:374)
	at net.minestom.server.entity.Entity.tick(Entity.java:543)
	at net.minestom.server.thread.TickThread.tick(TickThread.java:74)
	at net.minestom.server.thread.TickThread.run(TickThread.java:49)
```

*Version*:
- Using version: c5047b8037

**Comment by MasterBroNetwork** [comment]
Have you changed anything in Minestom's code?
Line 39 is a comment when checking the GitHub.

**Comment by TheMeinerLP** [comment]
@GoldenGamerLP There is a missing null check inside of the code. 
Ref: https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/listener/PlayerVehicleListener.java#L41

**Comment by MelonHell** [comment]
This has been fixed in 80f5db34a652e26004f33ab243008ff3a112eaa7

Judging by the stacktrace, such an exception simply cannot happen after this fix. I think the author used an older version than he indicated and this issue can be closed

**Comment by 36479756** [comment]
ok

> This has been fixed in [80f5db3](https://github.com/Minestom/Minestom/commit/80f5db34a652e26004f33ab243008ff3a112eaa7)
> 
> Judging by the stacktrace, such an exception simply cannot happen after this fix. I think the author used an older version than he indicated and this issue can be closed

ok
